pymongo/uri_parser.py

```diff
@@ -416,8 +417,13 @@ def parse_uri(uri, default_port=DEFAULT_PORT, validate=True, warn=False,
         scheme_free = uri[SCHEME_LEN:]
     elif uri.startswith(SRV_SCHEME):
         if not _HAVE_DNSPYTHON:
+            python_path = os.getenv('_', "python")
```

Comment:
Instead of `os.getenv` let's use `sys.executable` like this:
```python
python_path = sys.executable or 'python'
```

https://docs.python.org/3/library/sys.html#sys.executable

Reply:
Thanks for this!